docs/user-guides/workspace-access/cursor.md

```diff
@@ -0,0 +1,121 @@
+# Cursor
+
+[Cursor](https://cursor.sh/) is a modern IDE built on top of VS Code with enhanced AI capabilities.
+
+Follow this guide to use Cursor to access your Coder workspaces.
+
+If your team uses Cursor regularly, ask your Coder administrator to add a [Cursor module](https://registry.coder.com/modules/cursor) to your template.
+
+## Install Cursor and Coder CLI
```

Comment:
Installing Coder CLI is not required at all. Our extension takes care of this itself.